### PR TITLE
bug fix where LMS failed on cuda device. ones-array ref. grads to cas…

### PR DESCRIPTION
**Pull request by belsten**
…t to device. changed instantiation of coefficients to correct device instead of case. Avoids error with non-leaf autograd stuff

Also minor syntax changes/whitespace

### CODE REVIEW

File: sparsecoding/inference.py [hunk 1]
```diff
@@ -384,12 +387,18 @@ def infer(self, data, dictionary):
...
                 
                 # compute LSM loss for the current iteration
-                loss = self.lsm_Loss(data=data, dictionary=dictionary, coefficients=coefficients, lambdas=lambdas, sigma=self.sigma) 
+                loss = self.lsm_Loss(data=data, 
+                                     dictionary=dictionary, 
+                                     coefficients=coefficients, 
+                                     lambdas=lambdas, 
+                                     sigma=self.sigma
+                ) 
+                print(loss.device)
```
Comment (denkle) [inline sparsecoding/inference.py:396]: This print should probably be commented out

**Review by denkle — COMMENTED** [review]
Everything looks good except for a print statement

File: sparsecoding/inference.py (hunk 1 above)
Comment (belsten) [reply sparsecoding/inference.py:396]: oops thanks

**Comment by belsten** [comment]
Fixed!

**Comment by denkle** [comment]
Awesome!

**Pull request merged**